generators/client/templates/angular/src/main/webapp/app/shared/alert/alert-error.component.ts.ejs

```diff
@@ -87,7 +87,7 @@ export class AlertErrorComponent implements OnDestroy {
               this.addErrorAlert(`Error on field "${fieldName}"`<% if (enableTranslation) { %>, `error.${fieldError.message as string}`, { fieldName }<% } %>);
             }
           } else if (httpErrorResponse.error !== '' && httpErrorResponse.error.message) {
-            this.addErrorAlert(httpErrorResponse.error.message<% if (enableTranslation) { %>, httpErrorResponse.error.message, httpErrorResponse.error.params<% } %>);
+            this.addErrorAlert(httpErrorResponse.error.detail ?? httpErrorResponse.error.message<% if (enableTranslation) { %>, httpErrorResponse.error.detail ?? httpErrorResponse.error.message, httpErrorResponse.error.params<% } %>);
```

Comment:
```suggestion
            this.addErrorAlert(httpErrorResponse.error.detail ?? httpErrorResponse.error.message<% if (enableTranslation) { %>, httpErrorResponse.error.message, httpErrorResponse.error.params<% } %>);
```

With current patch the message will not be localized based on current user language.
From what I can see the problem only happens when i18n is disabled.
IMO with i18n, the message should be translated based on message and use detail as fallback if there are no translation for that message.

Reply:
>IMO with i18n, the message should be translated based on message and use detail as fallback if there are no translation for that message.

@mshima  Thanks for the review :) , for this case do you suggest checking if a translation exists for the message key using a `translateService.instant(message)!==message` and fallback to detail if it doesn't exist?

Reply:
Yep something like that.
```
const translatedMessage = translateService.instant(translationKey):
if (translatedMessage != translationKey) {
    alert.message = translatedMessage;
}
```
At https://github.com/jhipster/generator-jhipster/blob/22adabd1f9bf4c9562f723fb91045055d67123aa/generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs#L84

Reply:
Thanks for pointing that out.  For missing translations the `translateService.instant(translationKey)` returns `translation-not-found[translationKey]` string. It has now been handled in last commit.